README.md

```diff
@@ -16,7 +16,27 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
+```When you float an element, other elements will wrap around it, especially text. If you'd like to separate the floated element so that it has it's own line and nothing wraps around it you can use clearfix. This is one of a couple clean ways to clear the element.
+
+You would add the following css code for clearfix:
+
+.cf:before,
+.cf:after {
+    content: "Example text"; /* 1 */
+    display: table; /* 2 */
+}
+
+.cf:after {
+    clear: both;
+}
+
+/**
+ * For IE 6/7 only
+ * Include this rule to trigger hasLayout and contain floats.
+ */
+.cf {
+    *zoom: 1;
+}
 ```
 
```

Comment:
Good answer. Would also include that clearfix forces containers to (for lack of a better word) "contain" floated elements. Remember that example when the `<div>` collapsed because it only had a floated image inside of it?